crates/sui-config/src/node.rs

```diff
@@ -469,7 +469,7 @@ impl ExpensiveSafetyCheckConfig {
             force_disable_epoch_sui_conservation_check: false,
             enable_state_consistency_check: true,
             force_disable_state_consistency_check: false,
-            enable_move_vm_paranoid_checks: true,
+            enable_move_vm_paranoid_checks: false,
```

Comment:
Does this flag do anything anymore?

Reply:
It does not, but I kept the field present for backwards compatibility since this is a serialized data structure (and I believe part of the node config so is used quite a bit). We may be able to remove it, but I would need to see if removing it would break things...

Reply:
Looks like we can remove these fields!